i3_workspace_names_daemon.py

```diff
@@ -40,7 +40,10 @@ def build_rename(i3, app_icons, delim, length, uniq):
     """
     def get_icon_or_name(leaf, length):
         if leaf.window_class:
-            name = leaf.window_class
+            if leaf.window_class.lower() in match_instances:
```

Comment:
Do you know if  `leaf.window_class` can be `None`? If so can we add a guard here?